contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -181,9 +181,15 @@ script: |
     rm -rf ${DISTNAME}/lib/pkgconfig
     find ${DISTNAME}/bin -type f -executable -exec ../contrib/devtools/split-debug.sh {} {} {}.dbg \;
     find ${DISTNAME}/lib -type f -exec ../contrib/devtools/split-debug.sh {} {} {}.dbg \;
-    cp ../doc/README.md ${DISTNAME}/
     find ${DISTNAME} -not -name "*.dbg" | sort | tar --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ${OUTDIR}/${DISTNAME}-${i}.tar.gz
     find ${DISTNAME} -name "*.dbg" | sort | tar --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ${OUTDIR}/${DISTNAME}-${i}-debug.tar.gz
+    DOCDISTNAME="${DISTNAME}/share/doc/bitcoin"
+    mkdir -p ${DOCDISTNAME}/init
+    cp ../doc/README.md ${DOCDISTNAME}
```

Comment:
I think the goal of putting the README.md in the root was to make it easy to find for someone extracting it

If you move it to `share/doc/bitcoin`, well, you might as well make it part of the general `make install` logic in the build system.

Reply:
I included doc/README.md in the `make install` logic